packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -320,6 +320,10 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions) {
     usersWithCredentials = eventType.hosts.map(({ isFixed, user }) => ({ isFixed, ...user }));
   }
 
+  if (eventType.schedulingType && !eventType.hosts?.length) {
+    throw new TRPCError({ message: "No user is assigned.", code: "NOT_FOUND" });
+  }
+
```

Comment:
Earlier no slots would be displayed when no one was assigned to the team event type.

Reply:
There are slots displayed but when booking I am getting the error `Cannot read properties of undefined (reading 'name')`

Reply:
<img width="1512" alt="Screenshot 2024-02-28 at 10 22 17 AM" src="https://github.com/calcom/cal.com/assets/53316345/9d481aad-60d0-42c8-85e3-423acb25f37e">

@CarinaWolli if there are no hosts then you won't see any slots

Reply:
how can i reproduce the bug?

Reply:
> Earlier no slots would be displayed when no one was assigned to the team event type.

I meant before they were displayed because you said earlier there were no slots displayed. Now there are not